graylog2-server/src/main/resources/web-interface/index.html.template

```diff
@@ -12,7 +12,6 @@
   </head>
   <body>
     <script src="${appPrefix}/config.js"></script>
-    <script src="${appPrefix}/assets/vendor.js"></script>
```

Comment:
This produces a slight inconsistency between the production and development templates. Before the vendor script would always load before the polyfill chunk. Now that is only true in development.

I don't think loading the polyfill first would break anything, but I would keep the same loading order in both environments for consistency's sake.